### DISCUSSION TITLE
Search for a list of devices

### DISCUSSION THREAD

**Discussion by Surcof** [wmbusmeters/wmbusmeters#1272] [General]
I there, just purchrased my RTL2832U for start testing this world, i was wandering, if i kwon a list of devices that are in my flat, may i search for them untill i found all of them?
i try to explain better, is there a way to feed a list of devices (maybe by a .csv file) and start a search only for those and get notified when it finish?
Thank you in advice

**Comment by Big111156** [comment]
as in thh  README.md of the repository you can do this!..
"Check the contents of your /etc/wmbusmeters.conf file, assuming it has **device=auto:t1** and you are using a im871a,iu891a,amb8465(metis),amb3665,rc1180,cul or rtlsdr device, then you can now start the daemon with **sudo systemctl start wmbusmeters** or you can try it from the command line **wmbusmeters auto:t1**... "
